Documentation/Templates/ImportingTemplates.md

```diff
@@ -0,0 +1,28 @@
+# Using Advanced Editor for Importing Workbooks
+
+Azure Monitor Workbooks supports an advanced editor, where users create and modify workbooks in the JSON and ARM formats.
+
+From the Workbooks [GitHub template library](https://github.com/microsoft/Application-Insights-Workbooks/tree/templatedocs/Workbooks), Authors may wish to import specific Workbook JSON files into the portal.
+
+To do so, follow this step-by-step guide on how to access the Advanced Editor and upload your own Workbook file.
+
+1. Create a new empty workbook
```

Comment:
this should probably have step 1 be something like "open up workbooks from the place where the template is expected to be used. For example, if the template requires a Log Analytics Workspace, open the workbooks view from inside that workspace.  ..." something like that?  opening random templates might not work in random places.  (the workbook might parse just fine but not work if it expects to have a virtual machine as its "owner".   not sure how best to phrase that though?